api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java

```diff
@@ -268,6 +301,37 @@ private String getFreeCreditsResolutionText() {
         .build();
   }
 
+  private ImmutableMap<EmailSubstitutionField, String> setupBillingAccountEmailMap(
+      DbUser user, SendBillingSetupEmailRequest request) {
+
+    return new ImmutableMap.Builder<EmailSubstitutionField, String>()
+        .put(EmailSubstitutionField.HEADER_IMG, getAllOfUsLogo())
+        .put(EmailSubstitutionField.FIRST_NAME, user.getGivenName())
```

Comment:
sorry thinking more, these two should probably also be escaped, there is some potential there is some unsanitized content in here

Reply:
ah, okay. The reason I did not escape them is because those are all pulled from DbUser, not user arability input. 
But it is always good to add escape since we did not escape while user change their name in profile.